modules/core/shared/src/main/scala/lucuma/core/enum/MagnitudeBand.scala

```diff
@@ -65,4 +67,12 @@ object MagnitudeBand {
         MagnitudeBand.unsafeFromTag(s)
     }
 
+  /** @group Typeclass Instances */
+  implicit val MagnitudeBandOrder: Order[MagnitudeBand] =
+    Order.by(_.center)
```

Comment:
`Enumerated` extends `Order` of course so this sets up a situation where there are two implicit instances for `Order[MagnitudeBand]`.  Is it important that they be ordered by wavelength?  Maybe we should change the `Enumerated` instance such that the ordering is by wavelength?

Reply:
Good catch. I didn't think about it. I'd remove this, as the order is really needed only for `SortedMap`